src/devices/bresser_6in1.c

```diff
@@ -187,9 +187,9 @@ static int bresser_6in1_decode(r_device *decoder, bitbuffer_t *bitbuffer)
             "channel",          "",             DATA_INT,    chan,
             "battery_ok",       "Battery OK",   DATA_INT,    batt,
             "temperature_C",    "Temperature",  DATA_COND, temp_ok, DATA_FORMAT, "%.1f C", DATA_DOUBLE, temp_c,
-            "humidity",         "Humidity",     DATA_COND, temp_ok, DATA_INT,    humidity,
+            "humidity",         "Humidity",     DATA_COND, temp_ok && moisture < 0, DATA_INT,    humidity,
```

Comment:
Does this mean humidity is only shown when moisture < 0?

Reply:
`moisture >= 0` is valid and thus no humidity should be shown (the moisture level is transmitted in the humidity field)

Reply:
Ah ok, clear. Good to know :)